awx_collection/tests/integration/targets/tower_credential/tasks/main.yml

```diff
@@ -235,9 +382,10 @@
     kind: vault
   register: result
 
+# The creation of vault_cred_name2 never worked so we shouldn't actually need to delete it
 - assert:
     that:
-      - "result is changed"
+      - "result is not changed"
```

Comment:
I'd like @squidboylan to weigh in on whether we should delete that task or do this

Reply:
I'm fine leaving this in and if it gives us issues deleting it at that point